src/App.js

```diff
@@ -3,9 +3,32 @@ import { BrowserRouter, Routes, Route } from 'react-router-dom';
 import Landing from './components/Landing/Landing';
 import Store from './components/Store/Store';
 import Cart from './components/Cart/Cart';
+import GameDetails from './components/GameDetails/GameDetails';
 import './index.css';
 
 const App = () => {
+  const [wishList, setWishList] = React.useState([]);
+  const [showBlackScreen, setShowBlackScreen] = React.useState(false);
```

Comment:
the whole logic regarding this blackscreen should probably be extracted in a separate hook. E.g `useBlackScreen`

Reply:
Also [here](url) is a more elegant implementation of transitions between routes.